packages/create-cloudflare/e2e-tests/cli.test.ts

```diff
@@ -0,0 +1,31 @@
+import { execa } from "execa";
+import { describe, expect, test } from "vitest";
+import { version } from "../package.json";
+
+describe("Basic C3 functionality", () => {
+	const runCli = async (argv: string[]) => {
+		const result = await execa("node", ["./dist/cli.js", ...argv], {
+			stderr: process.stderr,
+		});
+
+		return result;
+	};
+
+	test("--version", async () => {
+		const result = await runCli(["--version"]);
+		expect(result.stdout).toContain(version);
```

Comment:
```suggestion
		expect(result.stdout).toEqual(`"${version}"`);
```

How about locking this down further?